app/models/move.rb

```diff
@@ -81,6 +81,10 @@ def cancel(reason: CANCELLATION_REASON_OTHER, comment: nil)
     )
   end
 
+  def self.unfilled?
+    pluck(:profile_id).any?(&:nil?)
```

Comment:
this is a bit inefficient, instead use the database with: `exists?(profile_id: nil)`

Reply:
ah nice this is exactly what I was looking for 👍